site/docs/tooltip.md

```diff
@@ -5,26 +5,30 @@ title: Tooltip
 permalink: /docs/tooltip.html
 ---
 
-Tooltip can provide details of a particular data point on demand. There are two ways to create a tooltip in Vega-Lite.
+Tooltip can provide details of a particular data point on demand. Tooltips are created with the `tooltip` channel. By default, the renderer with use native tooltips.
 
 ## Documentation Overview
 {:.no_toc}
 
 - TOC
 {:toc}
 
-## Using Tooltip channel
+## Tooltip channel
 
-To quickly create a tooltip without a plugin, Vega-lite's [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel can be mapped to a data field. For example, this bar chart supports tooltips for field `b`. Hover over the bar and notice the simple tooltip that displays the value of field `b` for each bar.
+To create a tooltip, Vega-Lite's [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel can be mapped to a data field. For example, this bar chart supports tooltips for field `b`. Hover over the bar and notice the simple tooltip that displays the value of field `b` for each bar.
 
 <div class="vl-example" data-name="bar_tooltip"></div>
 
-To show more than one field, you can calculate a new field that concatenates multiple fields. You can further customize the tooltip by specifying a custom event handler that gets invoked every time tooltip displays via [`tooltipHandler`](https://vega.github.io/vega/docs/api/view/#view_tooltipHandler)of the [`Vega View API`](https://vega.github.io/vega/docs/api/view/).
+To show more than one field, you can provide an array of fields or calculate a new field that concatenates multiple fields.
```

Comment:
> provide an array of fields

an array of field definitions

> or calculate a new field that concatenates multiple fields.

Not very clear from the text that this means literally using `calculate` transform?

Reply:
Fixed